ci/kind/kind-setup.sh

```diff
@@ -300,6 +300,18 @@ function configure_vlan_subnets {
       docker_run_with_host_net iptables -t filter -A FORWARD -i ${vlan_interfaces[j]} -o ${vlan_interfaces[i]} -j ACCEPT
     done
   done
+
+  # Adding ipset rules to prevent SNAT based on flexibleIPAM e2e tests
```

Comment:
remove this comment and add a more verbose and accurate one for each of the 2 iptables rules below.
In particular, this comment is not accurate for the second iptables rules.